content/tracing/advanced_usage/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Advanced APM Setup
+title: Advanced Languages instrumentation
```

Comment:
Can we call this just "Advanced instrumentation"? Seems like there's a lot on this page that doesn't pertain to languages. Also, it's nice to have the page heading match the navigation link.